src/Components/Patient/PatientHome.tsx

```diff
@@ -1059,7 +1059,7 @@ export const PatientHome = (props: any) => {
               {!patientData.present_health &&
                 !patientData.allergies &&
                 !patientData.ongoing_medication &&
-                patientData.gender === 2 &&
+                // patientData.gender === 2 &&
```

Comment:
You can not check `is_antenatal` before making sure that the patient is female. Hence, `patientData.gender === 2`  and `!patientData.is_antenatal` go together. Logically it's like "If Patient is not antenatal *provided* it's a female", So we only can test `is_antenatal` if we are sure that `gender` is `2`.

Related: https://github.com/coronasafe/care_fe/commit/8af43c29a1a59ee6ba3da94c6f3060546b9e1d51#diff-53fbb567d16de67259204bd83187f4916032665a0198c0f09fe3dd67a7541e6fR1062

```suggestion
                patientData.gender === 2 &&
```